cpp_test_suite/asyn/asyn_attr.cpp

```diff
@@ -65,7 +65,7 @@ int main(int argc, char **argv)
 				assert( db == 5.55 );
 				finish = true;
 			}
-			catch (AsynReplyNotArrived )
+			catch (AsynReplyNotArrived&)
```

Comment:
Should not be this **const** as well?

Reply:
You are right in principle but I think the current usage patterns tell us otherwise

```
$git grep "DevFailed" | grep catch | grep const | wc -l
21
firma@thomas:~/devel/cppTango (remove-some-compiler-warnings)
$git grep "DevFailed" | grep catch | grep -v const | wc -l
550
```

And if you look at the class structure of DevFailed/CorbaException there are no non-const member functions.

Reply:
Ok, got it! Thanks!